plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,790 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright John Berninger (@jberning) <john.berninger at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_backup
+short_description: Get, delete, create or update Proxmox VE backup jobs.
+version_added: 8.2.0
```

Comment:
```suggestion
version_added: 9.1.0
```

Reply:
Fix in 923fbd76014272b458eba79cdc257d9910718e23